processing/src/test/java/org/apache/druid/query/metadata/SegmentMetadataQueryTest.java

```diff
@@ -83,10 +83,15 @@
   public static QueryRunner makeMMappedQueryRunner(
       SegmentId segmentId,
       boolean rollup,
+      boolean bitmaps,
       QueryRunnerFactory factory
   )
   {
-    QueryableIndex index = rollup ? TestIndex.getMMappedTestIndex() : TestIndex.getNoRollupMMappedTestIndex();
+    QueryableIndex index = bitmaps
+                           ? rollup
```

Comment:
Nested ternaries = 😭. Please consider using normal `if`s.

Reply:
I know, these tests were already painful to slip in a test for this since parameterized, will see if I can clean up. Maybe removing size will help.